### PR TITLE
Feature/adding gesture protocol

### PR DESCRIPTION
**Pull request by bharathmurs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -819,6 +819,40 @@ public class YourCardPartsViewController: CardPartsViewController {
...
+`default is set to 1 second`.
+```swift
+    var minimumPressDuration: CFTimeInterval { get } // In seconds
+```
+
+Example:
+```swift
+extension MYOwnCardPartController: CardPartsLongGestureRecognizerDelegate {
+	func didLongPress(_ gesture: UILongPressGestureRecognizer) {
+		guard let v = gesture.view else { return }
```
Comment (croossin) [inline README.md]: Where do you use `v` after unwrapping?

Comment (croossin) [reply README.md]: Or are you just showing how you can get the gesture's view and do something??

Comment (bharathmurs) [reply README.md]: Just showing that could use v to animate :D

**Comment by croossin** [comment]
Tied to issue #45

**Pull request merged**